### DISCUSSION TITLE
Minimale Bodentemperatur in Kanal 8

### DISCUSSION THREAD

**Discussion by Hallodre** [OpenCCU/OpenCCU#1680] [🇩🇪 Fragen&Antworten] [answered]
Guten Morgen,

Seit einger Zeit betreibe ich eine CCU mit RaspberryMatic.
Da ich in einem Raum eine Feuerstelle habe und ein HomematicIP-Raumthermostat mit Schaltkontakt verbaut habe, wollte ich den Parameter "minimale Bodentemperatur" in Kanal 8 dazu nutzen, den Boden trotz überschrittener Lufttemperatur zu heizen, was aber augenscheinlich nicht möglich ist.
Wie ist dieser Parameter zu bewerten? Welche Funktion steckt dahinter?

Danke und VG

![image](https://user-images.githubusercontent.com/55655100/151111651-722ded8e-63ba-4809-98c4-1e01486990f6.png)

**Comment by Baxxy13** [comment, marked as answer]
Das Dokument: "WebUI - Parameterbeschreibungen" gibt dazu folgendes her:
>Ist ein Bodentemperatursensor im System vorhanden, gibt dieser Parameter an, welche Bodentemperatur mindestens gehalten werden soll. Mitunter bedeutet dies, dass weiter geheizt wird obwohl die Solltemperatur bereits erreicht wurde die Bodentemperatur aber nicht.

Ich würde die Fragen aber direkt an eQ-3 richten, die sollten ja am besten wissen was Sache ist.

**Reply by Hallodre** [reply]
Vielen herzlichen Dank.
Ich hab beim Support nachgefragt und dort wurde mir bestätigt, dass dieser Wert ein Fragment ist und nicht verwendet werden kann.

**Reply by alange6635** [reply]
Tolle Sache, wie kommt man nur darauf ELV (eQ-3) zu fragen?
